src/main/frontend/components/settings.cljs

```diff
@@ -93,14 +93,14 @@
        [:a.text-sm.fade-link.underline.inline
         {:target "_blank"
          :href "https://docs.logseq.com/#/page/changelog"}
-        "What's new?"]]]
+        (t :settings-page/changelog)]]]
 
      (when-not (or update-pending?
                    (string/blank? type))
        [:div.update-state.text-sm
         (case type
           "update-not-available"
-          [:p "Your app is up-to-date 🎉"]
```

Comment:
@sprocketc If you have opinions on translating emojis feel free to followup by mentioning it in the guide